### PR TITLE
Added notes to troubleshoot some issues with OpenGraph implementation

### PR DESCRIPTION
**Pull request by Jay-umbr**
I got someone who was very confused the siteName variable was not picked up... and I was too until I realized this guide is under Starter Kit and depends on its variables. Added some notes to make it more clear what might go wrong

### CODE REVIEW

File: Tutorials/Starter-kit/Lessons/3-Add-Open-Graph/step-4-v7.md [hunk 1]
```diff
@@ -61,6 +61,14 @@ The final view should look like this:
 <meta property="og:image" content="@Url.GetAbsoluteMediaUrl(Model.OpenGraphImage)" />
 ```
 
+:::note
+If you do not have Starter Kit installed, or experience issues with the properties not being picked up/showing up in HTML feel free to remove the fallback to siteName.
+:::
+
+:::note
+If your meta properties do not show up on social media, make sure to inspect source HMTL and make sure there are no inline HTML tags in og:title, og:description etc.
```
Comment (sofietoft) [inline Tutorials/Starter-kit/Lessons/3-Add-Open-Graph/step-4-v7.md:69]:
```suggestion
If your meta properties do not show up on social media, make sure to inspect source HMTL and make sure there are no inline HTML tags in `og:title`, `og:description` etc.
```

File: Tutorials/Starter-kit/Lessons/3-Add-Open-Graph/step-4.md [hunk 2]
```diff
@@ -73,6 +73,14 @@ var ogImage = Model.Value<IPublishedContent>("openGraphImage");
 <meta property="og:image" content="@ogImage.Url(mode: UrlMode.Absolute)" />
 ```
 
+:::note
+If you do not have Starter Kit installed, or experience issues with the properties not being picked up/showing up in HTML feel free to remove the fallback to siteName.
+:::
+
+:::note
+If your meta properties do not show up on social media, make sure to inspect source HMTL and make sure there are no inline HTML tags in og:title, og:description etc.
```
Comment (sofietoft) [inline Tutorials/Starter-kit/Lessons/3-Add-Open-Graph/step-4.md:81]:
```suggestion
If your meta properties do not show up on social media, make sure to inspect source HMTL and make sure there are no inline HTML tags in `og:title`, `og:description` etc.
```

**Review by sofietoft — CHANGES_REQUESTED** [review]
Thanks for adding the notes to this article @Jay-umbr - makes a lot of sense!

Perhaps consider adding them inside the same note, as it might be a bit too much with two boxes right after each other.
Also, I made a few suggestions.

**Comment by Jay-umbr** [comment]
> Thanks for adding the notes to this article @Jay-umbr - makes a lot of sense!
> 
> Perhaps consider adding them inside the same note, as it might be a bit too much with two boxes right after each other.
> Also, I made a few suggestions.

Good suggestion - updated so that these two notes are one note

**Review by sofietoft — APPROVED** [review]
Thanks for applying the suggesions @Jay-umbr !

Great work.
Let's get it merged! 💪

**Pull request merged**